pages/index.vue

```diff
@@ -191,8 +278,112 @@ export default Vue.extend({
   height: 91vh;
 }
 
+#location-detail-wrapper {
+  @apply m-0 relative;
+}
+
+#location-detail-card {
+  height: auto;
+  min-width: 574px;
+  font-family: "HYWenHei", sans-serif;
+  color: #484139;
+}
+
+#card-bg {
+  @apply absolute;
+  z-index: 5;
+  top: -3.75em;
+}
+
+#location-detail-card-content-wrapper {
+  @apply absolute flex right-0 mt-1 z-10;
+  height: 100%;
```

Comment:
尽量不要混用 `@apply` 和普通样式
另外这里可以直接写进 classname
```suggestion
  @apply absolute flex right-0 mt-1 z-10 h-full;
```

Reply:
啊那如果说有不得不有自己写的style，比如上个背景图片啥的，我就全写成普通的css还是？

Reply:
具体看需要写多少`@apply`多少手工style吧 如果可能的也许可以拆成两部份样式 :thinking: